### DISCUSSION TITLE
Not be able to see texts on the xgterm as they are small

### DISCUSSION THREAD

**Discussion by Lokhamb** [iraf-community/iraf#199] [Q&A] [answered]
If I type xgterm on the linux terminal it gives a message **xgterm:  unable to open font "lucidasanstypewriter-18", trying "fixed"....**
and the xgterm opens with small letters it is very very hard to make out. I have a 3.2k display on my laptop....[](url)
![Screenshot from 2021-10-16 19-25-51](https://user-images.githubusercontent.com/92636426/137590202-6627decf-8f25-4bf2-ae4a-f7282094c758.png)
![Screenshot from 2021-10-16 19-26-03](https://user-images.githubusercontent.com/92636426/137590204-62c40d18-1370-493c-a171-632de994c67e.png)

**Comment by olebole** [comment, marked as answer]
You could try to install the package `xfonts-100dpi`, which should contain this font.

**Comment by Lokhamb** [comment]
Now it is bigger a little bit....But is there any way to make it more big. I am new to iraf actually....
![Screenshot from 2021-10-16 20-42-52](https://user-images.githubusercontent.com/92636426/137592669-f13e727b-d907-47f5-a703-668a79a96371.png)

**Comment by olebole** [comment]
You could try out other sizes, as described on the web page: https://iraf-community.github.io/x11iraf.html
